x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/actions/connector_types/slack_api.ts

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import httpProxy from 'http-proxy';
+import expect from '@kbn/expect';
+import http from 'http';
+import getPort from 'get-port';
+import { getHttpProxyServer } from '@kbn/alerting-api-integration-helpers';
+import { getSlackServer } from '@kbn/actions-simulators-plugin/server/plugin';
+import { FtrProviderContext } from '../../../../../common/ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default function slackTest({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const configService = getService('config');
+
+  describe('Slack API action', () => {
```

Comment:
This will be interesting.  Because we've "hardcoded" the Slack API, we don't have an easy way to trick the test to use our own server, like we do with the other servers.  I think we'll have to do some kind of "sniff test" in the functional test server, and see if we're running FTS, and then don't use the hardcoded URL, but one that points back to the FTS, like the other simulators.

Maybe cleaner to have some new "internal" config setting we could use, in the `actions` plugin, to change the hardcoded URL to the server one.

Thinking we might be able to piggy back the new server simulator in with the existing one.  We can base it on whether the URL path is `/api` (slack api) or starts with `/services` (slack webhook).